container-runtime/src/Adapters/Systems/SystemForEmbassy/DockerProcedureContainer.ts

```diff
@@ -68,7 +68,10 @@ export class DockerProcedureContainer {
             },
           })
         } else if (volumeMount.type === "backup") {
-          throw new Error("TODO")
+          await overlay.mount(
+            { type: "volume", id: mount, subpath: null, readonly: false },
+            mounts[mount],
+          )
```

Comment:
Why not just create an effect to mount the backup to directly where it's wanted, instead of using a special path?